File: BubbleSort/BubbleSort.cs
Random rand = new Random();

int[] arr=new int[15];

for (int i = 0; i < arr.Length; i++)
{
    arr[i] = rand.Next(1, 100);
}
Console.WriteLine("\nArray Original: ");

for (int i = 0; i < arr.Length; i++)
{
    Console.Write(arr[i]+" ");
}

int mayor = 0;
for(int i = 0; i < arr.Length; i++)
{
    for(int j = 0; j < arr.Length - i - 1; j++)
    {
        if (arr[j] > arr[j + 1])
        {
            mayor = arr[j];
            arr[j] = arr[j + 1];
            arr[j + 1] = mayor;
        }
    }
}

Console.WriteLine("\nArray Ordenado: ");

for (int i = 0; i < arr.Length; i++)
{
    Console.Write(arr[i] + " ");
}

File: BubbleSort/BubbleSort.js
function random(min, max){
    return Math.floor(Math.random()*(max-min+1))+min;
}

let arr = new Array(15);

for(let i=0;i<arr.length;i++){
    arr[i]=random(1,100);
}
console.log("\nArreglo Original: ");

for(let i=0;i<arr.length;i++){
    console.log(arr[i])
}

let mayor=0;

for(let i=0;i<arr.length;i++){
    for(let j=0;j<arr.length-i-1;j++){
        if(arr[j]>arr[j+1]){
            mayor=arr[j]
            arr[j]=arr[j+1]
            arr[j+1]=mayor
        }
    }
}

console.log("\nArreglo Ordenado: ");

for(let i=0;i<arr.length;i++){
    console.log(arr[i])
}